solutions/platinum/cses-1191.mdx

```diff
@@ -124,5 +124,102 @@ int main() {
 	printf("%d\n", r);
 }
 ```
+
 </CPPSection>
+<PySection>
+
+<Warning>
+
+Due to Python's constant factor, the below solution TLEs on a couple test cases.
```

Comment:
This problem is doable in $O(N\log N)$ by computing binary lifting pointers and for each $i$, jumping until it reaches $i+n$.

Reply:
I can't see your solution, it's all blank

Reply:
I updated the c++ and py code along with the explanation for the updated solution which runs in `(n log n)` time.